Server/src/database.ts

```diff
@@ -16,5 +16,168 @@ export const connectDB = async () => {
+    let materialData2 = await connection.manager
+      .createQueryBuilder(Material, 'material')
+      .select('composition.composition', 'composition_composition')
+      .addSelect('material.details', 'material_details')
+      .innerJoin(Composition, 'composition', 'material.compositionId = composition.id')
+      .innerJoin('material.datasets', 'dataset')
+      .where('dataset.id = :datasetId', { datasetId: dataset2 })
+      .getRawMany();
+    console.log(materialData2);
+
+    console.log("Getting data point data #1");
+    let datapointData1 = await connection.manager
+      .createQueryBuilder(Datapoints, 'datapoints')
+      .select('datapoints.name', 'datapoints_name')
+      .addSelect('datapoints.values', 'datapoints_values')
+      .addSelect('units.units', 'units_units')
+      .addSelect('representations.repr', 'representations_repr')
+      .innerJoin(Units, 'units', 'datapoints.unitsId = units.id')
+      .innerJoin(Representations, 'representations', 'datapoints.representationsId = representations.id')
+      .innerJoin(Dataset, 'dataset', 'datapoints.datasetId = dataset.id')
+      .where('dataset.id = :datasetId', { datasetId: datasetReceived })
+      .getRawMany();
+    console.log(datapointData1);
+
+    console.log("Getting data point data #2");
+    let datapointData2 = await connection.manager
+      .createQueryBuilder(Datapoints, 'datapoints')
+      .select('datapoints.name', 'datapoints_name')
+      .addSelect('datapoints.values', 'datapoints_values')
+      .addSelect('units.units', 'units_units')
+      .addSelect('representations.repr', 'representations_repr')
+      .innerJoin(Units, 'units', 'datapoints.unitsId = units.id')
+      .innerJoin(Representations, 'representations', 'datapoints.representationsId = representations.id')
+      .innerJoin(Dataset, 'dataset', 'datapoints.datasetId = dataset.id')
+      .where('dataset.id = :datasetId', { datasetId: dataset2 })
+      .getRawMany();
+    console.log(datapointData2);
+
+
+    console.log("Getting data point comments for first data set");
+    let datapointComments1 = await connection.manager
+      .createQueryBuilder(Datapointcomments, 'datapointcomments')
+      .select('datapointcomments.comments', 'datapointcomments_comments')
+      .innerJoin(Dataset, 'dataset', 'datapointcomments.datasetId = dataset.id')
+      .where('dataset.id = :datasetId', { datasetId: datasetReceived })
+      .getRawMany();
+    console.log(datapointComments1);
+
+    console.log("Getting data point comments for second data set");
+    let datapointComments2 = await connection.manager
+      .createQueryBuilder(Datapointcomments, 'datapointcomments')
+      .select('datapointcomments.comments', 'datapointcomments_comments')
+      .innerJoin(Dataset, 'dataset', 'datapointcomments.datasetId = dataset.id')
+      .where('dataset.id = :datasetId', { datasetId: dataset2 })
+      .getRawMany();
+    console.log(datapointComments2);
```

Comment:
Are all of these queries needed? I dont think any of these belong in db.ts

Reply:
This is an unintentional merge conflict. These queries shouldn't be there; will fix